tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -541,6 +556,24 @@ class ScopedFilterDescriptor {
   SE_DISALLOW_COPY_AND_ASSIGN(ScopedFilterDescriptor);
 };
 
+// A helper class to decide whether to enable the TENSOR_OP_MATH math type
+template <bool DefaultFlag>
```

Comment:
This could be simply a function instead of a templated class:

```
  static bool TensorOpMathEnabled() {
    static bool is_enabled = [] {
      bool ret;
      tensorflow::ReadBoolFromEnvVar("TF_ENABLE_TENSOR_OP_MATH", /*default=*/true, &ret);
      return ret;
    }();
    return is_enabled;
  }
```

Reply:
Done